components/search_engines/brave_prepopulated_engines.cc

```diff
@@ -1,351 +1,81 @@
-  NULL,
-  NULL,
-  0,
-  SEARCH_ENGINE_OTHER,
-  PREPOPULATED_ENGINE_ID_WOLFRAMALPHA,
-};
-
-const PrepopulatedEngine yandex = {
-  L"Yandex",
-  L":ya",
-  "https://yandex.st/lego/_/rBTjd6UOPk5913OSn5ZQVYMTQWQ.ico",
-  "https://yandex.com/search/?text={searchTerms}&clid=2274777",
-  "UTF-8",
-  "https://suggest.yandex.com/suggest-ff.cgi?part={searchTerms}",
-  "https://yandex.com/images/search/?rpt=imageview",
-  NULL,
-  NULL,
-  NULL,
-  NULL,
-  NULL,
-  NULL,
-  "upfile={google:imageThumbnail},original_width={google:imageOriginalWidth},original_height={google:imageOriginalHeight},prg=1",
-  NULL,
-  0,
-  SEARCH_ENGINE_YANDEX,
-  15,
-};
-
-const PrepopulatedEngine youtube = {
-  L"YouTube",
-  L":yt",
-  "https://www.youtube.com/favicon.ico",
-  "https://www.youtube.com/results?search_type=search_videos&search_query={searchTerms}&search_sort=relevance&search_category=0&page=",
-  "UTF-8",
-  "https://suggestqueries.google.com/complete/search?output=chrome&client=chrome&hl=it&q={searchTerms}&ds=yt",
-  NULL,
-  NULL,
-  NULL,
-  NULL,
-  NULL,
-  NULL,
-  NULL,
-  NULL,
-  NULL,
-  0,
-  SEARCH_ENGINE_OTHER,
-  PREPOPULATED_ENGINE_ID_YOUTUBE,
-};
-
-}  // namespace TemplateURLPrepopulateData
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/search_engines/brave_prepopulated_engines.h"
+#include "components/search_engines/prepopulated_engines.h"
+#include "components/search_engines/search_engine_type.h"
+
+namespace TemplateURLPrepopulateData {
+
+// IMPORTANT! Make sure to bump this value if you make changes to the
+// engines below or add/remove engines.
+const int kBraveCurrentDataVersion = 3;
+
+const PrepopulatedEngine duckduckgo = {
+  L"DuckDuckGo",
+  L":d",
+  "https://duckduckgo.com/favicon.ico",
+  "https://duckduckgo.com/?q={searchTerms}&t=brave",
+  "UTF-8",
+  "https://ac.duckduckgo.com/ac/?q={searchTerms}&type=list",
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  0,
+  SEARCH_ENGINE_OTHER,
+  PREPOPULATED_ENGINE_ID_DUCKDUCKGO,
+};
+
+const PrepopulatedEngine qwant = {
+  L"Qwant",
+  L":q",
+  "https://www.qwant.com/favicon.ico",
+  "https://www.qwant.com/?q={searchTerms}&client=brz-brave",
+  "UTF-8",
+  "https://api.qwant.com/api/suggest/?q={searchTerms}&client=opensearch",
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  0,
+  SEARCH_ENGINE_OTHER,
+  PREPOPULATED_ENGINE_ID_QWANT,
+};
+
+const PrepopulatedEngine startpage = {
+    L"StartPage",
+    L":sp",
+    "https://www.startpage.com/favicon.ico",
+    "https://www.startpage.com/do/"
+    "dsearch?query={searchTerms}&cat=web&pl=opensearch",
+    "UTF-8",
+    "https://www.startpage.com/cgi-bin/"
+    "csuggest?query={searchTerms}&limit=10&format=json",
```

Comment:
some of these properties were originally full URLs: https://github.com/brave/brave-core/pull/1648/files#diff-7e5cc9096d62b4d079ea4a59f4d66d1bL229. did you mean to change them to paths instead of the full URL?

Reply:
This is due to lint complaining about 80-char line limits. The value is just broken into 2 lines.

Reply:
oops, misread this. my bad.

Reply:
As @AlexeyBarabash told in https://bravesoftware.slack.com/archives/C7VLGSR55/p1550090511069000, this fix makes code reading a little bit harder.
How about applying `//NOLINT` instead of splitting or using 4 indent in newline?